src/components/shared/CodeSection.tsx

```diff
@@ -33,9 +34,13 @@ export default function CodeSection(props: CodeSectionProps): JSX.Element {
 
   // Parameters for onClick are (addBattery, addSpeed, addStrength, addHunger)
 
-  const handleCorrect = (addBattery: number, addSpeed: number, addStrength: number, addHunger: number) => {
+  const handleCorrect = (addBattery: number, addSpeed: number, addStrength: number, addHunger: number, setChosenBox?: () => void) => { // eslint-disable-line
```

Comment:
Eslint says line is too long, automatically corrects it to one line when I split it up into multiple. Just set eslint to ignore on this one line